### PR TITLE
[one-cmds] Add tests for rawdata2hdf5

### PR DESCRIPTION
**Pull request by jinevening**
This adds tests for rawdata2hdf5

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #4944

### CODE REVIEW

File: compiler/one-cmds/tests/prepare_test_materials.sh
Comment (mhs4670go) [inline compiler/one-cmds/tests/prepare_test_materials.sh:55]: Just curiosity, is there any reason why you generates `preprocess.py` in script? It could be added as a python file.

Comment (jinevening) [reply compiler/one-cmds/tests/prepare_test_materials.sh:55]: No special reason. I put the code in this script simply because the name of the script is `prepare_test_materials.sh`.

If a separate file looks better, I'll move this code to the new file.

Comment (seanshpark) [reply compiler/one-cmds/tests/prepare_test_materials.sh:55]: If possible, can you embed the function of preprocess.py inside prepare_test_materials.sh ?
you can add the codes as a function if you like.

Comment (seanshpark) [reply compiler/one-cmds/tests/prepare_test_materials.sh:55]: maybe just provide `prepare_hdf5files.py` ?

Comment (mhs4670go) [reply compiler/one-cmds/tests/prepare_test_materials.sh:55]: @jinevening Well, it depends on your preference.
FYI, in my opinion, `cat` command with redirect uses when logging the outputs that is derived from this commands in our repo like this.
```bash
cat > "${PREFIX}.log" <(
      exec 2>&1

      rm -f "${PASSED_TAG}"

      set -ex
      # Create a pb model
      "${TFKIT_PATH}" encode "${TESTCASE_DIR}/test.pbtxt" "${PREFIX}.pb"

      # Do inference
      "${NNKIT_RUN_PATH}" \
        --backend "${NNKIT_TF_BACKEND_PATH}" \
        --backend-arg "${PREFIX}.pb" \
        --backend-arg "${TESTCASE_DIR}/test.info"
      set +ex

      touch "${PASSED_TAG}"
    )   
```
second, `.py` will be more readable in ide and make it easy to maintain and format check later.

Comment (jinevening) [reply compiler/one-cmds/tests/prepare_test_materials.sh:55]: > second, .py will be more readable in ide and make it easy to maintain and format check later.

It would be better to make a separate file. I'll move the code to a new file named `preprocess_images.py`.

CC @seanshpark

Comment (jinevening) [reply compiler/one-cmds/tests/prepare_test_materials.sh]: I've updated the code PTAL :)

**Comment by seanshpark** [comment]
Please do not change file attributes: `compiler/one-cmds/tests/prepare_test_materials.sh  100644 → 100755`
And please check for new files too.

**Comment by jinevening** [comment]
> Please do not change file attributes: compiler/one-cmds/tests/prepare_test_materials.sh 100644 → 100755

My mistake.. I've recovered the attribute. Thanks.

File: compiler/one-cmds/tests/preprocess_images.py
Comment (mhs4670go) [inline compiler/one-cmds/tests/preprocess_images.py:28]: Is it okay not to close after opening? With "with" block might helpful.

Comment (jinevening) [reply compiler/one-cmds/tests/preprocess_images.py]: I've updated to use 'with'. Thanks.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
I'll merge this after #4953 lands

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**
